File: AtCoder/Browns/python/ABC003-B-AtCoder_Trump.py
S = list(input())
T = list(input())
A = "atcoder"
win = "You can win"
lose = "You will lose"

for i in range(len(S)):
    if S[i] == T[i]:
        continue
    if T[i] == '@':
        S[i], T[i] = T[i], S[i]
    if S[i] != '@':
        print(lose)
        exit()
    f = False
    for a in A:
        if a == T[i]:
            f = True
            break
    if not f:
        print(lose)
        exit()
print(win)

File: AtCoder/Browns/cpp/ABC003-B-AtCoder_Trump.cpp
#include <bits/stdc++.h>
using namespace std;
typedef long long ll;

const string A = "atcoder";
const string win = "You can win";
const string lose = "You will lose";
string S, T;

int main() {
  cin >> S >> T;
  for (ll i = 0; i < S.size(); ++i) {
    if (S[i] != T[i]) {
      if (T[i] == '@') swap(S[i], T[i]);
      if (S[i] != '@') {
        puts(lose.c_str());
        return 0;
      }
      bool f = false;
      for (auto a : A) {
        if (a == T[i]) f = true;
      }
      if (!f) {
        puts(lose.c_str());
        return 0;
      }
    }
  }
  puts(win.c_str());
}